tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedReadHandleImpl.java

```diff
@@ -233,7 +242,10 @@ public static ReadHandle open(ScheduledExecutorService executor,
         // If we use a backoff to control the retry, it will introduce a concurrent operation.
         // We don't want to make it complicated, because in the most of case it shouldn't in the retry loop.
         while (retryCount-- > 0) {
+            long readIndexStartTime = System.nanoTime();
```

Comment:
We'd better record the retry times

Reply:
seems SNIP20 didn't mentioned it.

Reply:
The retry logic doesn't exist before snip 20 created